tests/test_calculator.py

```diff
@@ -1,10 +1,30 @@
 import unittest
 from calc.calculator import Calculator
 
-class TestCalculator(unittest.TestCase):
+
+class CalculatorTest(unittest.TestCase):
+    calculator = Calculator()
```

Comment:
Why do you need it here? There was no calculator instance in the previous file revision.

Reply:
Ok, my mistake. But anyway it doesn't matter